### ISSUE TITLE
How to deal with the loss backward function?

### ISSUE THREAD

**Issue by dongda118** [dontLoveBugs/DORN_pytorch#25]
Hi, is anyone melt with the problem about the loss.backward() function? in the original paper, it need calculated with x(w,h) and Pk, but i don't know which layer's output is the x(w,h) ? I don't find the code ,can you or anyone tell me how to deal with it? Thanks very much. and I hope to receive your answers.
